src/server.ts

```diff
@@ -29,7 +29,13 @@ import { Bit2MeService } from './services/bit2me.service';
 import { TiersService } from './services/tiers.service';
 import { MongoDBTiersRepository, TiersRepository } from './core/users/MongoDBTiersRepository';
 
-const start = async (mongoTestClient?: MongoClient): Promise<FastifyInstance> => {
+const start = async (
+  mongoTestClient?: MongoClient,
+  mocksServices?: {
```

Comment:
This a very bad idea. Tests are for covering the real codebase, you cannot mix both things. What you need to do here involves mechanisms like sinon's [stub](https://sinonjs.org/releases/latest/stubs/) method